### PR TITLE
AIR-859, LTS-1 security upgrades

### PR DESCRIPTION
**Pull request by Rohitrajak1807**
using image tag 0.72.1

### CODE REVIEW

**Comment by cruizen** [comment]
@Rohitrajak1807 can you attach the output of Trivy scan to get data on any open vulnerabilities?

File: trivy.report [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+│                   │                │          │                                              │                                     │ https://avd.aquasec.com/nvd/cve-2022-3294                    │
+└───────────────────┴────────────────┴──────────┴──────────────────────────────────────────────┴─────────────────────────────────────┴──────────────────────────────────────────────────────────────┘
+2023-04-11T06:52:37.450Z	INFO	Vulnerability scanning is enabled
+2023-04-11T06:52:37.681Z	INFO	Number of language-specific files: 1
+2023-04-11T06:52:37.681Z	INFO	Detecting gobinary vulnerabilities...
+2023-04-11T06:52:38.372Z	INFO	Vulnerability scanning is enabled
+2023-04-11T06:52:38.580Z	INFO	Number of language-specific files: 2
+2023-04-11T06:52:38.580Z	INFO	Detecting gobinary vulnerabilities...
+
+dikastes (gobinary)
```
Comment (cruizen) [inline trivy.report]: What binary is that? :-?

Comment (cruizen) [reply trivy.report]: It is a judge! https://en.wikipedia.org/wiki/Dikastes

**Comment by cruizen** [comment]
> @Rohitrajak1807 can you attach the output of Trivy scan to get data on any open vulnerabilities?

I see the Trivy report as one of the files in the changeset. Thanks. Added a link to that in the spreadsheet report we are sending.

**Comment by Rohitrajak1807** [comment]
Dikastes is built from the app-policy repo

**Comment by cruizen** [comment]
> > @Rohitrajak1807 can you attach the output of Trivy scan to get data on any open vulnerabilities?
> 
> I see the Trivy report as one of the files in the changeset. Thanks. Added a link to that in the spreadsheet report we are sending.

It will help if you run `trivy image <image URI>` and share that output for the (pf9) calico* images generated since we want to share that want info - instead of the output of `trivy fs .` on the repositories - let me know if we are already doing that.

**Comment by Rohitrajak1807** [comment]
> > > @Rohitrajak1807 can you attach the output of Trivy scan to get data on any open vulnerabilities?
> > 
> > 
> > I see the Trivy report as one of the files in the changeset. Thanks. Added a link to that in the spreadsheet report we are sending.
> 
> It will help if you run `trivy image <image URI>` and share that output for the (pf9) calico* images generated since we want to share that want info - instead of the output of `trivy fs .` on the repositories - let me know if we are already doing that.

we are already doing that. The scan now shows debian vulnerabilities as well...
might be due to a DB update. We're using the same flags for scanning. Check the latest report

File: trivy.report (hunk 1 above)
Comment (Rohitrajak1807) [reply trivy.report]: Comes from app policy repository

**Review by cruizen — APPROVED** [review]

**Pull request merged**